accepted/2020/workloads/workload-resolvers.md

```diff
@@ -0,0 +1,238 @@
+
+## Links
+
+- [Target Framework Names in .NET 5](https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md)
+- [NET SDK workloads overview](https://github.com/dotnet/designs/blob/workloads/accepted/2020/workloads/workloads.md) ([PR](https://github.com/dotnet/designs/pull/100))
+
+## Experiences
+
+- Projects using workloads just work if the workloads are installed
+- When opening a project in Visual Studio where the project depends on a workload that is not installed, Visual Studio will use the in-product acquisition experience to pop up a dialog box that notifies the developer what workloads are required, where they can click through to install the workload.
+- When building a project from the command line where the project depends on a workload that is not installed, a friendly error message will be generated specifying which workloads are required, and how to install them (ie by running the VS installer)
+- Improve existing experiences involving SDK resolvers
+  - Improve experience in VS when opening a project with a project.json that specifies an SDK that can't be found
+  - When using the NuGet package MSBuild SDK resolver, the UI should not hang while the NuGet package is acquired
+
+The following may not be delivered in .NET 5, but the design should support it:
+
+- A developer can run `dotnet bundle restore` in order to install all the workloads required by a project and its transitive project references.
+
+## Project files
+
+For .NET 5, iOS, Android, and WPF/Windows Forms project will all use the same `Microsoft.NET.Sdk` MSBuild SDK in the project file.  The `TargetFramework` will be used to specify the operating system, and a project targeting Windows will use the `UseWPF` or `UseWindowsForms` properties to opt in to WPF and/or Windows Forms.  For example, we plan for an Android project to look something like this:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0-android42</TargetFramework>
+  </PropertyGroup>
+
+</Project>
+```
+
+## Workload guidance
+
+The bulk of functionality from orkloads will be contained in workload "packs", which will be the unit of delivery for workloads.  This will include MSBuild targets and tasks.  The assets from the packs will be laid out under the dotnet root folder, probably in `packs/<Pack ID>/<Pack Version>` or a similar folder.
```

Comment:
```suggestion
The bulk of functionality from workloads will be contained in workload "packs", which will be the unit of delivery for workloads.  This will include MSBuild targets and tasks.  The assets from the packs will be laid out under the dotnet root folder, probably in `packs/<Pack ID>/<Pack Version>` or a similar folder.
```